game/src/devtools/fifteen_min/mod.rs

```diff
@@ -36,21 +36,27 @@ impl Viewer {
     pub fn new(ctx: &mut EventCtx, app: &App, start: BuildingID) -> Box<dyn State<App>> {
         let start = app.primary.map.get_b(start);
 
+        let title = Line("15-minute neighborhood explorer")
```

Comment:
Woops, please git pull and rebase -- we just moved the Panel construction into a separate method